conda/recipes/librmm/meta.yaml

```
@@ -85,25 +79,15 @@ outputs:
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
         - {{ compiler('cuda') }}
-        {% if cuda_major != "11" %}
-        - cuda-cudart-dev
-        {% endif %}
+        - {{ "cudatoolkit" if not (cuda_major == "12") else "cuda-version" }}
```

Sorry I had typoed this before

```suggestion
        - {{ "cuda-cudart-dev" if not (cuda_major == "11") else "cuda-version" }}
```